lib/evm/git.rb

```diff
@@ -8,8 +8,10 @@ def exist?
       File.exist?(@path)
     end
 
-    def clone(url)
-      git 'clone', url, @path
+    def clone(url, branch=nil)
+      args = 'clone', url, @path
+      args << "--branch=#{branch}" unless branch.nil?
```

Comment:
I would prefer this to avoid double negation:

```ruby
args << "--branch=#{branch}" if branch
```

Reply:
Done.